app/unitPages.rb

```diff
@@ -174,6 +170,7 @@ def getIssue(id)
 
 
 def unitSearch(params, unit)
+  # ToDo: remove this section now that unit search display looks just like global search
```

Comment:
It would be really nice to get rid of duplicative unitPages.unitSearch but I see series landing page needs it for sorting. Hrmmmm.